<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>494</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>486</width>
     <height>470</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>486</width>
     <height>470</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Задержки маркировки</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>92</height>
      </rect>
     </property>
     <property name="title">
      <string>Шаг прорисовки изображения</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Элементарный ход ЦАПа. Лазер включен</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxDXYLaserOn">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>23</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>56</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Элементарный ход ЦАПа. Лазер выключен</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxDXYLaserOff">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>20000</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>461</width>
       <height>224</height>
      </rect>
     </property>
     <property name="title">
      <string>Задержки вывода изображения</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка на вывод одной точки (dot delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPointDelay">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>25</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>57</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка вектора маркировки (mark delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>52</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>84</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка между векторами маркировки (polygon delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>79</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>106</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>111</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка в конце перехода (jump delay)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>138</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка в конце перехода (min jump delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>133</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>165</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка включения лазера (laser on delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxLaserOnDelay">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>160</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>192</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Задержка выключения лазера (laser off delay)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxLaserOffDelay">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>187</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>343</y>
       <width>461</width>
       <height>86</height>
      </rect>
     </property>
     <property name="title">
      <string>Q-SW</string>
     </property>
     <widget class="QSpinBox" name="spinBoxQSWPeriod">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>22</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>27</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Период Q-SW (мкс)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Длительность Q-SW (мкс)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxQSWDuration">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>49</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <attribute name="title">
     <string>Настройки программы</string>
    </attribute>
    <widget class="QGroupBox" name="groupScreenOutputWay">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>10</y>
       <width>465</width>
       <height>151</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Способ вывода на экран</string>
     </property>
     <widget class="QRadioButton" name="radioFastDraw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Быстрый вывод</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioLaserEmulation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Эмуляция работы лазера</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupLaserEmulationWay">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>38</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioSteps">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Шаги</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioLines">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Линии</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>70</y>
        <width>71</width>
        <height>4</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxDrawIdleMotion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Выводить серыми линиями холостой ход лазера</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupScreenOutputWay_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>170</y>
       <width>465</width>
       <height>61</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Коррекции</string>
     </property>
     <widget class="QCheckBox" name="checkCorrTurnOn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>27</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Включить коррекцию точек, если файл коррекций corr.asc найден</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBoxDXYLaserOn</tabstop>
  <tabstop>spinBoxDXYLaserOff</tabstop>
  <tabstop>spinBoxPointDelay</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBoxLaserOnDelay</tabstop>
  <tabstop>spinBoxLaserOffDelay</tabstop>
  <tabstop>spinBoxQSWPeriod</tabstop>
  <tabstop>spinBoxQSWDuration</tabstop>
  <tabstop>radioFastDraw</tabstop>
  <tabstop>radioLaserEmulation</tabstop>
  <tabstop>radioLines</tabstop>
  <tabstop>radioSteps</tabstop>
  <tabstop>checkBoxDrawIdleMotion</tabstop>
  <tabstop>checkCorrTurnOn</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
